frontend/src/routes/Applications/Overview.test.tsx

```diff
@@ -235,29 +239,37 @@ describe('Applications Page', () => {
     userEvent.click(screen.getByRole('button', { name: /close openshift/i }))
   })
 
-  test.skip('export button should produce a file for download', async () => {
-    nockAggegateRequest('applications', fetchAggregate.req, applicationAggregate.res)
+  test('export button should produce a file for download', async () => {
+    nockAggegateRequest('applications', fetchAggregate.req, fetchAggregate.res)
 
     await waitForText('feng-remote-argo8')
 
     window.URL.createObjectURL = jest.fn()
     window.URL.revokeObjectURL = jest.fn()
-    const documentBody = document.body.appendChild
-    const documentCreate = document.createElement('a').dispatchEvent
 
-    const anchorMocked = { href: '', click: jest.fn(), download: 'table-values', style: { display: '' } } as any
-    const createElementSpyOn = jest.spyOn(document, 'createElement').mockReturnValueOnce(anchorMocked)
-    document.body.appendChild = jest.fn()
-    document.createElement('a').dispatchEvent = jest.fn()
+    const { blobConstructorSpy, createElementSpy } = getCSVExportSpies()
 
-    userEvent.click(screen.getByLabelText('export-search-result'))
+    userEvent.click(screen.getByTestId('export-search-result'))
     userEvent.click(screen.getByText('Export all to CSV'))
 
-    expect(createElementSpyOn).toHaveBeenCalledWith('a')
-    expect(anchorMocked.download).toContain('table-values')
-
-    document.body.appendChild = documentBody
-    document.createElement('a').dispatchEvent = documentCreate
-    await new Promise((resolve) => setTimeout(resolve, 1500))
+    await waitFor(() => {
+      const toastElement = screen.getByText(/Export successful/i)
+      expect(toastElement).toBeInTheDocument()
+    })
```

Comment:
@KevinFCormier `frontend/src/routes/Applications/Overview.test.tsx` was the problematic test case. I found that, for some reason the test was ending before the document used its "createElement" object for the csv file. I added a waitFor and looked out for the success toast alert. That did it.